solr/modules/s3-repository/build.gradle

```diff
@@ -20,70 +20,70 @@ apply plugin: 'java-library'
 description = 'S3 Repository'
 
 dependencies {
-    api project(':solr:core')
-    implementation project(':solr:solrj')
+  api project(':solr:core')
+  implementation project(':solr:solrj')
 
-    implementation 'org.apache.lucene:lucene-core'
+  implementation 'org.apache.lucene:lucene-core'
```

Comment:
if we make this "api" on solr-core, then would we _still_ need to declare this here?

Reply:
That is 100% a valid question and one I don't know the answer to. I think there are a few other implementation -> api that we could make at `core`, `test-framework` and maybe a few others that would make the downstream consumers dependency declaration a bit cleaner. I could try it out, but didn't want to make too many unnecessary changes here.